pkg/controller/utils/clientwrapper.go

```diff
@@ -89,15 +98,18 @@ type ControllerMetricsTripper struct {
 
 // RoundTrip implements the http RoundTripper interface.
 func (cmt *ControllerMetricsTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	startTime := time.Now()
 	remoteStr := "false"
 	if cmt.Remote {
 		remoteStr = "true"
 	}
 	path, pathErr := parsePath(req.URL.Path)
 	// Call the nested RoundTripper.
 	resp, err := cmt.RoundTripper.RoundTrip(req)
+	applyTime := metav1.Now().Sub(startTime).Seconds()
 	if err == nil && pathErr == nil {
 		metricKubeClientRequests.WithLabelValues(cmt.Controller, req.Method, path, remoteStr, resp.Status).Inc()
+		metricKubeClientRequestSeconds.WithLabelValues(cmt.Controller, req.Method, path, remoteStr, resp.Status).Observe(applyTime)
```

Comment:
This is showing up weirdly fast, like 0.012s for a remote GET, 0.009 for a POST, is this in the right place? @staebler